rfd/0138/README.md

```diff
@@ -0,0 +1,146 @@
+---
+authors: Rui Loura <rui@joyent.com>
+state: predraft
+discussion: 
+---
+
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+<!--
+    Copyright 2018 Joyent, Inc.
+-->
+
+# RFD 138 Multi-subnet Admin Networks
+
+Today, the admin network consists of a single subnet over which all of the services across each host and rack communicate.  The goal of this change is to increase scalability and reduce size of L2 domain, by changing to a Clos (spine leaf) topology.  The result of this change will be support for a multi-subnet admin networks so that each rack will be on its own L2 admin broadcast domain, and rack-to-rack (cross subnet) traffic will be done via L3 switching.  
```

Comment:
I would say this more like:

> The goal of this change is to allow multiple admin subnets so that deployments can scale their number of CNs while keeping a reduced L2 domain, and allow for Clos (spine leaf) network topologies.

I would also add at the end of the paragraph:

> Triton and Manta administration tools will be updated to allow provisioning to an `"admin"` network pool which aggregates all of the admin subnets.